java/base-client/src/main/java/no/bankaxept/epayment/client/base/RequestResponse.java

```diff
@@ -0,0 +1,34 @@
+package no.bankaxept.epayment.client.base;
+
+import java.util.Objects;
+
+public class RequestResponse<T> {
+    private final RequestStatus status;
+    private final T response;
```

Comment:
I think RequestResponse sounds a bit weird. But I also understand why it is named like that. Could it however be named like Response, with Status status and T body? Or something similar?

Reply:
We don't need the class at all, re. my [other comment](https://github.com/techcloud0/bankaxept-epayment-development-kit/pull/16#discussion_r847165606).